hardware/MQTT.cpp

```diff
@@ -3076,19 +3134,11 @@ void MQTT::InsertUpdateSwitch(_tMQTTASensor* pSensor)
 						szOnOffValue = "Set Level";
 						// recalculate level to make relative to min/maxpositions
 						if (pSensor->component_type == "cover") {
-							if (
```

Comment:
Could you undo the formatting here. It makes the code harder to read

Reply:
I don't know, why this was changed, I did not do it knowingly at least. Anyway, I will change it and also insert a Cr before the opening brace on the line before to bring it in line with he rest of the coding style.